packages/office-ui-fabric-react/src/utilities/grid/Grid.base.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  getId,
+  toMatrix,
+  classNamesFunction
+} from '../../Utilities';
+import { FocusZone } from '../../FocusZone';
+import { IGrid, IGridProps, IGridStyleProps, IGridStyles } from './Grid.types';
+
+const getClassNames = classNamesFunction<IGridStyleProps, IGridStyles>();
+
+export class GridBase extends BaseComponent<IGridProps, {}> implements IGrid {
+
+  private _id: string;
+
+  constructor(props: IGridProps) {
+    super(props);
+    this._id = getId();
+  }
+
+  public render() {
+    let {
+      items,
+      columnCount,
+      onRenderItem,
+      positionInSet,
+      setSize,
+      getStyles
+    } = this.props;
+
+    const classNames = getClassNames(getStyles!);
```

Comment:
pretty sure you should at least be passing in theme here in case someone wants to style the grid with a palette color.

Reply:
Will do 👍